sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -662,6 +691,24 @@ public partial class CapacityPoolPatch : Azure.ResourceManager.Models.TrackedRes
         public static bool operator !=(Azure.ResourceManager.NetApp.Models.EnableNetAppSubvolume left, Azure.ResourceManager.NetApp.Models.EnableNetAppSubvolume right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct FileAccessLog : System.IEquatable<Azure.ResourceManager.NetApp.Models.FileAccessLog>
```

Comment:
This is kind of a generic name - could we rename this to something like `NetappFileAccessLog`